### ISSUE TITLE
Constant CS0436 warning

### ISSUE THREAD

**Issue by M4przybysz** [seblj/roslyn.nvim#349]
### Description

I'm new to neovim and I tried to set it up as an external code editor for Unity. I managed to do most of the things and I'm using Roslyn as C# LSP and there's a problem. I'm constantly getting CS0436 warning when referencing my MonoBehaviour classes as if they were conflicting with themselves (they aren't). I didn't find any way to disable this warning except for "#pragma". 

Example of this warning:
The type 'GameManager' in '<project_directory>\Assets\Scripts\GameManager.cs' conflicts with the imported type 'GameManager' in 'Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'. Using the type defined in '<project_directory>\Assets\Scripts\GameManager.cs'. [CS0436]

Replaced sensitive data with "<project_directory>" and "\<user>"

### Neovim version

NVIM v0.12.1

### Operating system and version

Windows 10

### Expected behavior

No warning when referencing C# classes.

### Actual behavior

Warning CS0436 when referencing classes. Unity doesn't show any warnings but Roslyn does.

Example:
The type 'GameManager' in '<project_directory>\Assets\Scripts\GameManager.cs' conflicts with the imported type 'GameManager' in 'Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'. Using the type defined in '<project_directory>\Assets\Scripts\GameManager.cs'. [CS0436]

### Minimal config

```Lua
I've installed LazyVim distro, roslyn-nvim and nvim-unity-cync.
I'm using Unity 6.3 (6000.3.10f1) with Neovim Code Editor (com.apyra.nvim-unity) package.
```

### Roslyn checkhealth

```text
==============================================================================
roslyn:                                                                     ✅

roslyn.nvim: Requirements ~
- ✅ OK Neovim >= 0.12
- ✅ OK dotnet SDK >= 10 (found 10.0.202)

roslyn.nvim: Roslyn Language Server ~
- ✅ OK C:/Users/<user>/AppData/Local/nvim-data/mason/bin/roslyn.cmd: found

roslyn.nvim: Razor extension ~
- ✅ OK Razor extension: found at C:/Users/<user>/AppData/Local/nvim-data/mason/packages/roslyn/libexec/.razorExtension
- ✅ OK vscode-html-language-server: found
- ✅ OK html-lsp client: configured

roslyn.nvim: Roslyn extensions ~

roslyn.nvim: 'razor' ~
- ✅ OK Enabled: true
- ✅ OK Resolved path: 'C:/Users/<user>/AppData/Local/nvim-data/mason/packages/roslyn/libexec/.razorExtension/Microsoft.VisualStudioCode.RazorExtension.dll' (file exists)
- ✅ OK Resolved args:
  --razorSourceGenerator=C:/Users/<user>/AppData/Local/nvim-data/mason/packages/roslyn/libexec/.razorExtension/Microsoft.CodeAnalysis.Razor.Compiler.dll
  --razorDesignTimePath=C:/Users/<user>/AppData/Local/nvim-data/mason/packages/roslyn/libexec/.razorExtension/Targets/Microsoft.NET.Sdk.Razor.DesignTime.targets

roslyn.nvim: File Watching Configuration ~
- File watching: auto (using Neovim's file watcher)

roslyn.nvim: Solution Detection ~
- ✅ OK Selected solution: <project_directory>/Projekt_Cyfrowy3.sln
```

**Comment by seblyng** [comment]
Like an actual diagnostic? If so, you can use .editorconfig file for example to disable differenet diagnostic codes. My guess is that this has nothing to do with roslyn.nvim if so

**Comment by M4przybysz** [comment]
Yeah. Roslyn somehow sees any reference to a class I created and that is in file referenced in Assembly-CSharp.csproj as a conflict and shows a CS0436 warning. 
Thankfully .editorconfig with code below inside solved the issue.
```
root = true

[*.cs]
dotnet_diagnostic.CS0436.severity = none
```
